docs/en/ingest-management/elastic-agent/otel-agent-configuration.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[otel-agent-configuration]]
+= {agent} OTel configuration
+
+preview::[]
+
+When running in `otel` mode, {agent} supports a standard OTel Collector configuration. 
+
+You can provide the OTel configuration to {agent} using any of the following four schemes:
+
+* `file` - The configuration is read from a file. For example, `file:path/to/config.yaml`.
+* `env` - The configuration is read from an environment variable. For example, `env:MY_CONFIG_IN_AN_ENVVAR`.
+* `yaml` - The configuration is read from a standard YAML-formatted file. For example, `yaml:exporters::debug::verbosity: detailed`.
+* `http` - The configuration is read from an HTTP URL. For example, `http://www.example.com`.
+// Perhaps we can improve these examples to make them more clear. I'm not sure I understand what the configuration would look like when passed as a URL, for example.
+
+To help you get started, when you download an {agent} install package, the package comes with a sample OTel configuration file, `otel.yml` located in the package base directory.
```

Comment:
is the file below the otel.yml from the distro?

Reply:
Yes, the below file content matches what I can see in `otel.yml` from the Elastic Agent `v8.14.0` download.

Not sure how we can make sure the below blob is kept up to date?

Reply:
> is the file below the otel.yml from the distro?

Yup. I've updated it to include the Extensions section which is new in the [8.15 version](https://github.com/elastic/elastic-agent/blob/main/otel.yml).

> Not sure how we can make sure the below blob is kept up to date?

Indeed! We currently don't have a way to embed repo code content directly in the docs. So our options are:
 
. We include a copy of the configuration in the docs. I can update it when the configuration changes provided that the code owners tag me on the PRs so that I know when this happens.
. We don't include the configuration in the docs, and just have a link to it, something like `You can always find the latest configuration file at <URL>`.